modules/op-release-notes-1-12.adoc

```diff
@@ -0,0 +1,177 @@
+
+* With this update, you can use parameters under the `platforms.openshift.pipelinesAsCode.options.configMaps.pac-config-cogging.data` spec to set logging levels for components of {pac}.
+
+* With this update, you can set policies that allow certain actions only to members of a team and reject the actions when other users request them. Currently, the `pull_request` and `ok_to_test` actions support setting such policies.
+
+* With this update, you can pass arbitrary parameters in the incoming webhook as a JSON payload. {pipelines-shortname} passes these parameters to the pipeline run. To provide an additional security layer, you must explicitly define the permitted parameters in the `Repository` CR.
+
+* With this update, matching a large set of pipeline runs with a large number of remote annotations in {pac} is optimized. {pac} fetches the remote tasks only for the pipeline run that has matched.
+
+* With this update, you can use the `source_url` variable in a pipeline run template to retrieve information about the forked repository from which the event, such as a Pull Request or Push, is coming.
++
+.Example usage
+[source,yaml]
+----
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+# ...
+spec:
+  params:
+    - name: source_url
+      value: "{{ source_url }}"
+  pipelineSpec:
+    params:
+      - name: source_url
+# ...
+----
+
+* With this update, if an authorized user provides an `ok-to-test` comment to trigger a pipeline run on the pull request from an unauthorized user and then the author makes further changes to the branch, {pac} triggers the pipelines. To disable triggeting the pipeline until an authorized user provides a new `ok-to-test` comment, set the `pipelinesAsCode.settings.remember-ok-to-test` spec in the `TektonConfig` CR to false.
+
+* With this update, on the GitHub status check page, the table that shows the status of all tasks includes the display name of every task.
+
+* With this update, you can configure the `tags push` event in a pipeline run on GitLab.
+
+* Release Notes: With this update, you can use the `target_url` and `source_url` fields in {pac} Common Expression Language (CEL) expression filtering annotations to filter the request for specific target or source.
+
+* With this update, when configuring fetching a remote GitHub URL using a token, you can include a branch name that contains a slash. You must encode the slash within the branch name as `%2F` to ensure proper parsing by {pac}m as in the following example URL: `https://github.com/organization/repository/blob/feature%2Fmainbranch/path/file`. In this example URL, the branch name is `feature/mainbranch` and the name of the file to fetch is `/path/file`.
+
+* In this release, you can use `--v1beta1` flag in the `tkn pac resolve` command. Use this flag if the pipeline run is generated with the `v1beta1` apiVersion schema.
+
+[id="breaking-changes-1-12_{context}"]
+== Breaking changes
```

Comment:
Please add one more breaking change:
TektonResult CR no longer supports the following fields in the spec:

```
  db_user: 
  db_password: 
  s3_bucket_name: 
  s3_endpoint: 
  s3_hostname_immutable: 
  s3_region: west
  s3_access_key_id: 
  s3_secret_access_key: 
  s3_multi_part_size: 
```

Please add them via secret. Please remove the existing TektonResult CR and recreate it after installation.